src/main/java/de/bonndan/nivio/assessment/AssessmentRepository.java

```diff
@@ -0,0 +1,47 @@
+package de.bonndan.nivio.assessment;
+
+import de.bonndan.nivio.model.FullyQualifiedIdentifier;
+import de.bonndan.nivio.model.Landscape;
+import org.springframework.lang.NonNull;
+import org.springframework.lang.Nullable;
+import org.springframework.stereotype.Component;
+
+import java.util.HashMap;
+import java.util.Objects;
+
+/**
+ * Stores and loads and creates Assessments with FullyQualifiedIdentifier as key
+ */
+@Component
+public class AssessmentRepository {
+
+    private final HashMap<FullyQualifiedIdentifier, Assessment> repository;
+
+    public AssessmentRepository() {
+        repository = new HashMap<>();
+    }
+
+    public void clean() {
+        repository.clear();
+    }
+
+    public Assessment createAssessment(@NonNull Landscape landscape) {
```

Comment:
Public methods, especially in a widely used factory should have javadoc explaining what is happening.